parlai/crowdsourcing/tasks/fast_acute/conf/example.yaml

```diff
@@ -0,0 +1,7 @@
+#@package _global_
```

Comment:
If it's possible, I'd expect that the `example.yaml` file is one that would run off-the-bat for someone to see what a run looks like without needing to do configuration. Something like this I think would be more a `copy_me.yaml` or similar, but that's just my thought.

Reply:
Yes, good point - changing